### ISSUE TITLE
SelectOneMenu: For Editable="true", ItemValue not working. Item lable getting displayed instead of ItemValue.

### ISSUE THREAD

**Issue by ShivaniSK** [primefaces/primefaces#3731]
## 1) Environment
- PrimeFaces version: 6.1

## 2) Expected behavior
 SelectOneMenu should store itemValue. 
...

## 3) Actual behavior
SelectOneMenu storing itemLabel. With editable="true" attribute it is not working. If i remove it then it works. I want it to work for editable=true also.
..

## 4) Steps to reproduce
Select any value from dropDown.  The userID value should be displayed in the input textBox.
.. 

## 5) Sample XHTML
```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:ui="http://java.sun.com/jsf/facelets"
	xmlns:h="http://java.sun.com/jsf/html"
	xmlns:p="http://primefaces.org/ui"
	xmlns:f="http://java.sun.com/jsf/core">
	<h:head>

	<meta charset="utf-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge" />	
	<script src="../js/projectRequest.js"></script>
</h:head>	
	<h:form id="tst">

		
	<p:selectOneMenu id="city" value="#{projectBean.requestorID}" effect="fold" editable="true" 
	required="true" requiredMessage="City is required."> 
            <f:selectItem itemLabel="Select One" itemValue="" />
            <f:selectItems value="#{projectBean.users}" var="user" itemValue="#{user.userID}" itemLabel="#{user.firstName}"  />
             <p:ajax event="change" update="cityValue" />
   </p:selectOneMenu>
   
   <br/> <br/>
   
   <p:inputText value = "#{projectBean.requestorID}" id="cityValue"></p:inputText>
   
   
   
	</h:form>
</html>

```
..

## 6) Sample bean
```
import javax.faces.bean.ViewScoped;
import javax.faces.bean.ManagedBean;
import javax.annotation.PostConstruct;

@ManagedBean(name = "projectBean")
@ViewScoped
public class ProjectBeanCompUI {

	@PostConstruct
	public void init()
	{
         UserInfo userInfo = new UserInfo();
    	 userInfo.setFirstName("Shivani");
    	 userInfo.setLastName("Kanakhara");
    	 userInfo.setUserID("skanakh");
    	 UserInfo userInfo2 = new UserInfo();
    	 userInfo2.setFirstName("Sravani");
    	 userInfo2.setLastName("Bandaru");
    	 userInfo2.setUserID("banmani");
    	   
    	 users.add(userInfo);
    	 users.add(userInfo2);
    	   
}

private List<UserInfo> users = new ArrayList<>();
private String requestorID;
public String getRequestorID() {
		return requestorID;
	}
	public void setRequestorID(String requestorID) {
		this.requestorID = requestorID;
	}
	public List<UserInfo> getUsers() {
		return users;
	}
	public void setUsers(List<UserInfo> users) {
		this.users = users;
	}
}



public class UserInfo{
    
    private String firstName;
    private String lastName;
    private String userID;

     public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

     public String getUserID() {
		return userID;
	}

	public void setUserID(String userID) {
		this.userID = userID;
	}

}
```
..

**Comment by melloware** [comment]
I think this was already fixed in PF 6.2.   Verify and close please.
See this ticket:  https://github.com/primefaces/primefaces/issues/2862

**Comment by ShivaniSK** [comment]
Issue resolved in 6.2. Thanks
